R/create_dummy_data_e1d.R

```diff
@@ -39,6 +39,7 @@ create_dummy_data_e1d <- function(df_type = "student") {
                                                     size = 100,
                                                     replace = TRUE),
                          StudentLevel = c(rep("Undergraduate", 40), rep("Graduate", 40), rep("Undergraduate", 20)),
+                         IsHighSchool = c(rep(0, 99), 1),
```

Comment:
I'm assuming we're adding this flag in the dummy data so that we can calculate dual enrollment?